doc/development/guide/tests.rst

```diff
@@ -0,0 +1,25 @@
+Software tests
+==============
```

Comment:
While on this page, the `Contents` section appears on the right empty. Is there a way to either add a small content section/remove it easily?